### DISCUSSION TITLE
Deprecated Elastic Versions Don't allow elastic agent enrollment

### DISCUSSION THREAD

**Discussion by nightstalker117** [Security-Onion-Solutions/securityonion#13976] [2.4] [answered]
### Version

2.4.110

### Installation Method

Security Onion ISO image

### Description

upgrading

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

6

### RAM

22

### Storage for /

5GB

### Storage for /nsm

2TB

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

I am currently running Elastic 8.14.3 inside SO.  Elastic 8.15.1 is the minimum required version for being able to successfully deploy the elastic agent on MacOS 15.1.1. When will the upgrade for this be put into soup?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by defensivedepth** [comment]
We have macs on 15.1.1 that are running with the current version of Security Onion, which uses Elastic 8.14.3.

Where are you seeing that 8.15.1 is the minimum required version?

**Reply by advxrsary** [reply]
I am not the author of the thread, but I am looking for an upgrade to >8.15.1 as well, as it is the minimum required version of the agent for Elastic Defend to work on Mac OS Sequoia (15.x). 

Ref:
https://discuss.elastic.co/t/macos-sequoia-15-x-support/364235

**Reply by nightstalker117** [reply]
@advxrsary @defensivedepth  This is the exact issue that I am talking about.

**Comment by defensivedepth** [comment]
@nightstalker117 @advxrsary So you can install the Agent, just not use Elastic Defend.

We are planning on upgrading Elastic components in the near future.

**Reply by nightstalker117** [reply]
Correct.

**Reply by reyesj2** [reply, marked as answer]
2.4.130 was just released and includes ES 8.17.3 which should resolve this issue 
https://github.com/Security-Onion-Solutions/securityonion/discussions/14376
